pkg/controller/servicemesh/memberroll/namespace_reconciler.go

```diff
@@ -26,6 +26,7 @@ import (
 
 const networkTypeOpenShiftSDN = "OpenShiftSDN"
 const networkTypeCalico = "Calico"
+const networkTypeNsxNcp = "ncp"
```

Comment:
This should be a more descriptive value.  `ncp` would have no meaning to most users who are reviewing/attempting to understand a configuration file with a value like this.

Reply:
Thanks for reviewing @brianredbeard ! This is the value the certified cluster operator watches in the network crd during openshift bootstrapping. It means NSX Container Plugin.  Users that have NSX installed and plan to use it for Openshift know this abbreviation. Furthermore, we have already customers and changing this value might trigger issues in their clusters during upgrade. Please accept the constant name a more descriptive here.

Reply:
I understand the name might not tell much, but it's unfortunately the name of the networking solution "NSX Container Plugin" as Yasen wrote.